coordinator/src/main/java/org/apache/uniffle/coordinator/ApplicationManager.java

```diff
@@ -70,6 +70,7 @@ public ApplicationManager(CoordinatorConf conf) {
   public void refreshAppId(String appId) {
     if (!appIds.containsKey(appId)) {
       CoordinatorMetrics.counterTotalAppNum.inc();
+      LOG.info("Newly registered application: {}", appId);
```

Comment:
how about `New application is registered: {}`?

Reply:
It's OK for me. Done